src/resources/ParentedResource.ts

```diff
@@ -0,0 +1,32 @@
+import Resource from './Resource';
+
+export default class ParentedResource<R, T extends R> extends Resource<R, T> {
+  protected defaultParentId: string | undefined;
+
+  /**
+   * Returns the passed parent identifier, or `defaultParentId` as set by `withParent` if the former is `undefined`.
+   */
+  protected getParentId(input: string | undefined): string | undefined {
+    if (input != undefined) {
+      return input;
+    } /* if (input == undefined) */ else {
+      return this.defaultParentId;
```

Comment:
At Mollie, we prefer as little `else` statements as possible. We can simply return instead of going through the `else` :-)

Reply:
Also, I would actually turn statements like these around. If undefined, return the default parent ID.

Reply:
I remember now that we had this discussion before! I'll try and internalise this style.